filebeat/check.py

```diff
@@ -0,0 +1,59 @@
+# (C) Datadog, Inc. 2010-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# stdlib
+
+# 3rd party
+import json
```

Comment:
nitpick but this should be using simplejson instead (way better perf)

Reply:
Sure :)